.github/workflows/test-cache-hexpm.yml

```diff
@@ -0,0 +1,34 @@
+---
+name: test-cache-hexpm
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  cache_hexpm:
+    name: Cache Hex.pm
+    runs-on: ${{matrix.os}}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ['ubuntu-latest', 'windows-latest']
```

Comment:
This will eventually fail once we reach a point where OTP 24 is not supported for `-latest`. I'd prefer a fixed version, that we can later revisit.